packages/@sanity/base/components.d.ts

```diff
@@ -1 +1,3 @@
-export * from './src/components'
+import * as components from './src/components'
+
+export = components
```

Comment:
Why is this needed? Won't this require the consuming end to have `esModuleInterop` enabled?

Reply:
It's using the `export =` because that's how to define CommonJS export (afaik). See the `components.js` that this def is referring to:

```js
module.exports = require('./lib/components')
```